diagnostic.md

```diff
@@ -6,72 +6,91 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+To store data and to send that data to the client upon requests.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+the model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+the database
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+Because we have the controller skip that step by sending JSON back to the client;
+it is more efficient that way.
 ```
 
 What does C.R.U.D stand for?
 
 ```bash
-// your response here
+Create Read Update Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```bash
-// your response here
+the controller
 ```
 List at least 5 standard actions that C.R.U.D corresponds to?
 
 ```bash
-// your response here
+Create - Create
+Read - index, show
+Update - Patch
+Destroy - Delete
 ```
 
 A user action fires a `GET` request for `person/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```bash
-// your response here
+<ol>
+  <li>The GET request gets sent to the router.</li>
+  <li>The router takes the GET request and the given URL and matches it to the
+  correct controller action.</li>
+  <li>That controller action communicates to the appropriate model which houses the
+  business logic</li>
+  <li>The model communicates with the database and the databse returns the data back to the model.</li>
+  <li>The model gives that data to the controller and the controller turns the data into JSON and sends it to the client.</li>
+</ol>
 ```
```

Comment:
lol, I didn't mean you have to write and ordered list in HTML, justs "-" or something